source/_docs/energy/electricity-grid.markdown

```diff
@@ -59,6 +63,38 @@ _Attention! Installing CT clamp sensor devices requires opening your electrical
 
 Some energy providers will provide you real-time information about your usage and have this data integrated into Home Assistant.
 
+### Manual Integration
```

Comment:
This page provides general information on energy integration. The example provided below is very specific for MQTT which IMHO doesn't belong on this page.

Reply:
Example removed.